### DISCUSSION TITLE
How to resume training with different instance images without ruining the already trained instance before?

### DISCUSSION THREAD

**Discussion by cdefghijkl** [TheLastBen/fast-stable-diffusion#1142] [General]
Good morning.

I have a question.

A few hours ago, I resumed my last trained model which had one female face "'A'" trained. It's working great.
Then I resume training after I tested it for a while with another instance images which is different female "B" with the same setting as before but I added some concept images which the images nothing similar to "A" and "B" images, and I remove the last instance images which had "A" images because I'm worried it will be training it again.

After done, I tested it again with the same trigger "A" filename in the prompts, but it ruined the "A" look which is the similarity of the face is now more like "B" face.

Is there any way I can resume the first training to train "B" face without ruining "A" face similarity that has been trained?

**Comment by TheLastBen** [comment]
That is called overfitting it's the main issues in this field when training on a small dataset.

**Reply by cdefghijkl** [reply]
It's not because the "B" overwrite the "A" data?

**Reply by TheLastBen** [reply]
It doesn't completely overwrites A, it messes with the common weights like (human, person ...etc)

**Reply by Aninda165** [reply]
Would the solution would be training a new checkpoint for each new face?

**Reply by TheLastBen** [reply]
that would be ideal

**Reply by Aninda165** [reply]
Thanks for the quick response

**Reply by Aninda165** [reply]
Would training multiple faces at one go help? Trying to cut down on the number of individual checkpoints as each checkpoint is 2-4 GB

**Reply by TheLastBen** [reply]
it could work, but the genders must be different, if you train two males, the faces could merge
